src/ansi-c/scanner.l

```diff
@@ -1281,6 +1281,7 @@ __decltype          { if(PARSER.cpp98 &&
 {CPROVER_PREFIX}"finally"      { loc(); return TOK_CPROVER_FINALLY; }
 {CPROVER_PREFIX}"ID"           { loc(); return TOK_CPROVER_ID; }
 {CPROVER_PREFIX}"loop_invariant" { loc(); return TOK_CPROVER_LOOP_INVARIANT; }
+{CPROVER_PREFIX}"loop_variant" { loc(); return TOK_CPROVER_LOOP_VARIANT; }
```

Comment:
Are you committed to the name?  My concern is that if we have `loop_invariant` and `loop_variant` it is natural to think that they are duals / inverses which is not really the intuition we want.  Might it be better to call them ranking functions or similar?

Reply:
Thanks, this is a good point.
There are several options here. For example, Dafny calls it `decreases` (and I think ACSL too). Coq calls it `measures`.
I personally think `__CPROVER_decreases` would be the clearest.

What do you think?

cc: @kroening @tautschnig

Reply:
For the syntax of loop variants, I've swithced from `__CPROVER_loop_variant(...)` to `__CPROVER_decreases(...)`. If we change our mind later, we can easily change the syntax.